alarm/nialarm.yml

```diff
@@ -0,0 +1,686 @@
+        take:
+          description: Number of entries to include in the response list. Typically used in combination with the skip parameter to implement pagination.
+          type: integer
+          example: 10000
+          default: 10000
+          maximum: 10000
+
+responses:
+  Error:
+    description: Error
+    schema:
+      description: Error response
+      title: ErrorResponse
+      type: object
+      properties:
+        error:
+          $ref: '#/definitions/Error'
+  QueryResponse:
+    description: Query response
+    schema:
+      description: Query response
+      title: QueryResponse
+      type: object
+      properties:
+        totalCount:
+          description: The total number of alarm instances which matched your query
+          type: integer
+          example: 1
+        filterMatches:
+          description: Array containing the alarm instances which matched your query
+          type: array
+          items:
+            $ref: '#/definitions/AlarmInstance'
+  Unauthorized:
+    description: Not authorized
+    headers:
+      WWW_Authenticate:
+        description: Information for how to authenticate
+        type: string
+paths:
+  /:
+    get:
+      tags: [versioning]
+      summary: API information
+      description: Returns information about API versions and available operations.
+      operationId: RootEndpoint
+      x-ni-request-all-privileges: true
+      # Explicitly mark security as an empty array - this route does not require any privileges.
+      # Marking it this way prevents it from inheriting the top-level security settings.
+      security: []
+      responses:
+        200:
+          description: OK
+          schema:
+            description: Version information
+            title: RootEndpointResponse
+            type: object
+            properties:
+              v1:
+                $ref: '#/definitions/V1Operations'
+              version:
+                description: Implementation version of the web service
+                type: string
+  /{version}:
+    parameters:
+      - in: path
+        name: version
+        description: Version of the API to retrieve operations.
+        type: string
+        required: true
+    get:
+      tags: [versioning]
+      summary: API version information
+      description: Returns available operations for a single version of the API.
+      operationId: RootEndpointWithVersion
+      x-ni-request-all-privileges: true
+      # Explicitly mark security as an empty array - this route does not require any privileges.
+      # Marking it this way prevents it from inheriting the top-level security settings.
+      security: []
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/V1Operations'
+        404:
+          description: Not Found
+          schema:
+            $ref: '#/definitions/Error'
+  /v1/acknowledge-instances:
+    post:
+      tags: [alarmInstances]
```

Comment:
Tags can contain spaces and should be all lowercase, so `alarm instances` would be fine.

Reply:
Nice, I much prefer them that way. For some reason I thought past guidance had been to camelCase them.